pom.xml

```diff
@@ -123,16 +113,16 @@
                                     <version>1.3</version>
                                 </additionalDependency>
                             </additionalDependencies>
-                            <additionalparam>
-                                -hdf book.path /docs/reference/_book.yaml
-                                -hdf project.path /_project.yaml
-                                -hdf devsite.path /docs/reference/admin/java/reference/
-                                -d ${project.build.directory}/apidocs
-                                -templatedir ${devsite.template}
-                                -toroot /docs/reference/admin/java/reference/
-                                -yaml _toc.yaml
-                                -warning 101
-                            </additionalparam>
+                            <additionalOptions>
+                                <additionalOption>-hdf book.path /docs/reference/_book.yaml</additionalOption>
```

Comment:
We need to pass the `-Xdoclint:none` option here too. Earlier it was provided by the profile whenever JDK 1.8 was available. Now we have to explicitly set it.

Reply:
Thanks! Added `<doclint>none</doclint>`